joplin/templates/wagtailadmin/login.html

```diff
@@ -1,3 +1,11 @@
 {% extends "wagtailadmin/login.html" %}
+{% load render_bundle from webpack_loader %}
 
-{% block branding_login %}Welcome to Joplin!{% endblock %}
+{% block branding_login %}
+{% if ISPRODUCTION == False %}
```

Comment:
render 'theme' bundle (which is literally just an import of a scss file atm, but could be expanded in the future)